### PR TITLE
fix: get funding via docs, not via bzzaar

### PR DESCRIPTION
**Pull request by Eknir**
Don't refer users immediately to the bzzaar, but refer to the docs (where users find where to go to).

---
This change is [Reviewable](https://reviewable.io/reviews/ethersphere/bee/1746)

### CODE REVIEW

**Review by acud — APPROVED** [review]

File: pkg/settlement/swap/chequebook/init.go [hunk 1]
```diff
@@ -73,7 +73,7 @@ func checkBalance(
 				logger.Warningf("cannot continue until there is at least %d BZZ available on %x", neededERC20, overlayEthAddress)
 			}
 			if chainId == 5 {
-				logger.Warningf("get your Goerli ETH and Goerli BZZ now via the bzzaar at https://bzz.ethswarm.org/?transaction=buy&amount=%d&slippage=30&receiver=0x%x", neededERC20, overlayEthAddress)
+				logger.Warningf("get your Goerli ETH and Goerli BZZ now via https://docs.ethswarm.org/docs/installation/fund-your-node")
```
Comment (acud) [inline pkg/settlement/swap/chequebook/init.go:76]: `learn how to fund your node by visiting our docs at https://...`

File: packaging/homebrew/bee-get-addr [hunk 2]
```diff
@@ -13,7 +13,7 @@ After you fix configuration run 'bee-get-addr' again.
         ETH_ADDRESS=$(echo "$RESP" | grep ethereum | cut -d' ' -f6 | tr -d '"')
         echo "
 Please make sure there is sufficient eth and bzz available on $ETH_ADDRESS address.
-You can get both Goerli ETH and Goerli BZZ now via the bzzaar at https://bzz.ethswarm.org/?transaction=buy&amount=10&slippage=30&receiver=0x$ETH_ADDRESS
+You can get both Goerli ETH and Goerli BZZ now via the bzzaar at https://docs.ethswarm.org/docs/installation/fund-your-node
```
Comment (acud) [inline packaging/homebrew/bee-get-addr:16]: learn how to fund your node by visiting our docs at.... the same goes everywhere

Comment (Eknir) [reply packaging/homebrew/bee-get-addr:16]: Ok, better.

Comment (Eknir) [reply packaging/homebrew/bee-get-addr]: addressed

**Review by vandot — APPROVED** [review]

File: packaging/homebrew/bee-get-addr (hunk 2 above)
Comment (ldeffenb) [reply packaging/homebrew/bee-get-addr]: Well, https://docs.ethswarm.org/docs/installation/fund-your-node gives me:

Page Not Found
We could not find what you were looking for.

Please contact the owner of the site that linked you to the original URL and let them know their link is broken.

Hopefully that doesn't get missed before the release!

**Pull request merged**